cdk/package.json

```diff
@@ -11,30 +11,30 @@
 		"test:dev": "jest --watch",
 		"format": "prettier --write \"{lib,bin}/**/*.ts\"",
 		"cdk": "cdk",
-		"lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
+		"lint": "eslint lib/**/*.ts bin/**",
 		"synth": "cdk synth --path-metadata false --version-reporting false",
 		"diff": "cdk diff --path-metadata false --version-reporting false"
 	},
 	"devDependencies": {
-		"@guardian/cdk": "^58.1.3",
+		"@guardian/cdk": "^58.1.4",
 		"@guardian/eslint-config-typescript": "^11.0.0",
-		"@guardian/prettier": "^8.0.0",
-		"@guardian/tsconfig": "^0.2.0",
+		"@guardian/prettier": "^8.0.1",
+		"@guardian/tsconfig": "^1.0.0",
 		"@types/jest": "^29.5.12",
 		"@types/node": "20.14.10",
-		"aws-cdk": "2.148.0",
-		"aws-cdk-lib": "2.148.0",
+		"aws-cdk": "2.149.0",
+		"aws-cdk-lib": "2.149.0",
 		"constructs": "10.3.0",
-		"eslint": "^8.56.0",
+		"eslint": "^9.7.0",
 		"jest": "^29.7.0",
-		"ts-jest": "^29.1.5",
+		"ts-jest": "^29.2.2",
 		"ts-node": "^10.9.2",
-		"tslib": "^2.6.2",
+		"tslib": "^2.6.3",
 		"typescript": "~5.5.3"
 	},
 	"dependencies": {
-		"npm": "^10.4.0",
-		"source-map-support": "^0.5.20"
+		"npm": "^10.8.2",
```

Comment:
Do we need `npm` as a dependency if the `cdk` directory is using `pnpm`?

Reply:
Removed